server/src/core/index.js

```diff
@@ -1,7 +1,7 @@
 
 
 const setup = async () => {
-    const logger = require('pino')()
+    const logger = require('./logger');
```

Comment:
Thanks a lot for your PR. here is a suggestion. 
please do a conditional logger selection.
eg. 
```
const logOption = process.env.logger;
if(logOption==='pino') return createPino() 
return createWinston();
```

Reply:
added the conditional logger selection on `logger.js` file. Kindly check that.

Reply:
thanks. :)